index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>New document11111</title>
+    <link rel="stylesheet" href="styles/style.css" type="text/css"/>
+    <title>Lesson 3</title>       
 </head>
 <body>
+       <script src="./index.js"></script>
+</body>
+</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Tables</title>
+</head>
+<body>
+    <header>
+        I'm headerrrrrrrrrr
+    </header>
+    <p> Какой-то текст с объяснением, <mark> почему</mark> я выбрал FE.Так почему же?<br> </font> 
+        Ну, во первых - потому что FE сейчас популярен и в HTML/css можно въехать только упорством и усидчивостью. <em>Далее я снова пишу набор букв что бы абзац получился побольше
+        пока непонятно как задать размер шрифта. Надо уже использовать CSS? 
+    <span>&#128522;</span>
+        <p class="symbone">Двойная стрелка вправо</p>
+</p>
+    <p>Дорогий дневник, нужно продолжать что-то <del>писать</del> сочинять, но я уже не в силах шевелить мозгами и просто добавляю теги для домашки один за одним. </p>
+    Кстати, пора примернить тег sub, но что я могу <sub>написать</sub> что бы его использовать?
+    <span>&#128521;</span>
+</p>
+<p class="symbtwo">случайный символ</p>
+     
+    <img class="first-pic" src="/images/fox.png" alt="Текст для картинки">
```

Comment:
the image is not loading, pls change your src to "./images/fox.png" or "images/fox.png"